ui/pages/providers/add/details.js

```diff
@@ -99,8 +99,6 @@ export default function ProvidersAddDetails() {
           </label>
           <input
             required
```

Comment:
this doesn't show the 1Password in the input container?

Reply:
Great catch!! Will revert this